gravitee/values.yaml

```diff
@@ -151,17 +165,21 @@ ui:
     enabled: true
     # Used to create an Ingress record.
     hosts:
-      - developer.example.com
+      - apim.example.com
     annotations:
       kubernetes.io/ingress.class: nginx
       kubernetes.io/app-root: /management
       kubernetes.io/rewrite-target: /management
       ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\n"
```

Comment:
Do you know if there is any way to enable gzip encoding for JS media-type by default ?

Reply:
I found a way, but we need to add another file to /etc/nginx/conf.d/
and set with:

```
gzip on;
gzip_proxied any;
gzip_types text/plain text/xml text/css application/x-javascript;
gzip_vary on;
gzip_disable "MSIE [1-6]\.(?!.*SV1)";
```
It will gzip compress the types we list in `gzip_types`

If you want, I can add it to [gravitee/templates/ui-configmap.yaml](https://github.com/gravitee-io/gravitee-kubernetes/blob/fix-helm-template/gravitee/templates/ui-configmap.yaml#L11) and submit another PR.

Reply:
Yes, it would be perfect because the size of the JS is really lower than the original one and it is really interesting for users with a low bandwith (like me ;) ).

We also missed that point from our docker-compose for the API platform...
https://github.com/gravitee-io/gravitee-docker/blob/master/platform/nginx/nginx.conf